src/components/templates/PartyMap/PartyMapAdmin.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState, useEffect, FC, useCallback } from "react";
+import { useSelector } from "hooks/useSelector";
+import firebase from "firebase/app";
```

Comment:
If this is for getting a reference to firebase, there is a `useFirebase()` hook which is probably a better 'style'.